insights/parsers/messages.py

```diff
@@ -86,3 +86,23 @@ def get(self, s):
                     msg_info['procname'] = info_splits[4]
                 r.append(msg_info)
         return r
+
+    def get_after(self, timestamp, lines=None):
+        """
+        Get a list of lines after the given time stamp.
+
+        Parameters:
+            timestamp(datetime.datetime): log lines after this time are
+                returned.
+            lines(list): the list of log lines to search (e.g. from a get).
+                If not supplied, all available lines are searched.
+
+        Returns:
+            (list): The list of log lines with time stamps after the given
+                date and time.
+
+        *NOTE*: Because /var/log/messages timestamps by default have no year,
+        you must supply the year in your timestamp as 1900.  This may also
+        cause confusion around the end of the year.
```

Comment:
Sorry, what I really should have said is how should the messages parser handle the case of where log messages go from Dec 31 to Jan 1 if we are hard coding the year?  Should the parser do something to handle that?